bin/setupReplication.js

```diff
@@ -100,12 +117,24 @@ class _SetupReplication {
         this._sourceBucket = sourceBucket;
         this._targetBucket = targetBucket;
         // TODO: support bootstrap list failover
-        const destEndpoint = destination.bootstrapList[0].servers[0];
+        const destEndpoint = '217.182.11.96:8000';
```

Comment:
When building Federation client-template, I did not set `destination.bootstrapList` so I hard-coded this value in for testing. I think this is set in `env/client-template/group_vars/all` using `env_replication_endpoints`?